spec/controllers/users_controller_spec.rb

```diff
@@ -73,5 +73,49 @@
         end
       end
     end
+
+    describe 'GET #search' do
+      before(:each) do
+        @alaine = User.create(
+          username: 'alaine',
+          email: 'alaine@live.com',
+          password: 'alaine',
+          password_confirmation: 'alaine'
+        )
+        @alaine.confirm!
+
+        @alen = User.create(
+          username: 'alenesteibar',
+          email: 'alenesteibar@live.com',
+          password: 'alen',
+          password_confirmation: 'alen'
+        )
+        @alen.confirm!
+      end
+
+      context 'when the user is logged in' do
+        it 'responds successfully with an HTTP 200 status code' do
+          get :search, term: "est"
+          expect(response).to be_success
+          expect(response.code.to_i).to eq(200)
+        end
+
+        it 'returns a json with the matching users' do
```

Comment:
use https://relishapp.com/rspec/rspec-core/docs/pending-and-skipped-examples/skip-examples#skipping-using-`skip` so you remember to fix it later :D